entity-management-web/src/main/java/eu/europeana/entitymanagement/config/AppConfig.java

```diff
@@ -72,5 +76,15 @@ public ValidatorFactory getValidatorFactoryBean() {
     public EuropeanaClientDetailsService getClientDetailsService() {
 	return new EuropeanaClientDetailsService();
     }
-    
+
+
+    /**
+     * Configures Spring Batch to use Mongo
+     * @param datastore Morphia datastore for Spring Batch
+     * @return BatchConfigurer instance
+     */
+    @Bean
```

Comment:
please assign a bean name

Reply:
In this case, a bean name is unnecessary as:

-  there will only ever be one `BatchConfigurer` within the application
-  this bean is loaded by the Spring context (so we never get to use its qualifier)